src/features/defi/helpers/utils.ts

```diff
@@ -12,3 +16,17 @@ export const chainIdToLabel = (chainId: ChainId): string => {
     }
   }
 }
+
+export const makeHasEnoughBalanceForGas = (
```

Comment:
You're right the terminology might be a bit confusing, `make` is typically used as an higher-order-function to make functions
`getHasEnoughBalanceForGas` is definitely cleaner, probably still the same issue as my naming though, which is prepending something to `hasEnoughBalanceForGas` to avoid naming conflicts, I guess we could probably take the problem the other way around and name this function what we're really trying to get as a boolean.

I think "canCoverTxFee" would be a good heuristic, matching the intent of that method, and also matching the standard "Not enough ETH for transaction fee" message